export-and-sign/src/event-handlers.js

```diff
@@ -376,16 +385,12 @@ async function createSolanaKeypair(privateKey) {
  */
 function validateKey(key, keyAddress) {
   if (!key) {
-    throw new Error(
-      `key bytes not found. Please re-inject export bundle for address ${keyAddress} into iframe. Note that address is case sensitive.`
-    ).toString();
+    throw new Error(getKeyNotFoundErrorMessage(keyAddress)).toString();
   }
 
   const now = new Date().getTime();
   if (now >= key.expiry) {
-    throw new Error(
-      `key bytes not found. Please re-inject export bundle for address ${keyAddress} into iframe. Note that address is case sensitive.`
-    ).toString();
+    throw new Error(getKeyNotFoundErrorMessage(keyAddress)).toString();
```

Comment:
can we use a separate message to indicate that it's because the key has expired?